### ISSUE TITLE
New release timeline?

### ISSUE THREAD

**Issue by stylesuxx** [vendidero/woocommerce-germanized#190]
Hey, I am wondering when there will be a new release - master has quite some things fixed that are not in 3.13.2 - we have product bundles with virtual products and our current installation is throwing errors and we can't sell product for over a week. I was thinking to deploy the master state but I am not fully aware of the potential side effects and wanted to know if there is a timeline for the next release?

**Comment by dennisnissle** [comment]
Currently, master does only contain minor fixes. Guess you are using an outdated bundles version (< 6.0) if you are receiving errors? Or what is the actual error thrown? 

It should be enough to replace the bundles compatibility script with the master version: https://github.com/vendidero/woocommerce-germanized/blob/master/includes/compatibility/class-wc-gzd-compatibility-woocommerce-product-bundles.php

We do not yet have a specific timeline for our next release.

Best

**Comment by stylesuxx** [comment]
Thanks for they quick response.

I am getting the following error:
```
PHP Fatal error:  Uncaught Error: Call to undefined method WC_Product_Bundle::is_virtual_bundle() in /xx/xxx/[xxxx/wp-content/plugins/woocommerce-germanized/includes/compatibility/class-wc-gzd-compatibility-woocommerce-product-bundles.php:70](http://xxx/wp-content/plugins/woocommerce-germanized/includes/compatibility/class-wc-gzd-compatibility-woocommerce-product-bundles.php:70)
```
I tried replacing that file but other errors pop up where similar safe-guards are in place.

`Woocommerce Product Bundles` is actually on 6.1.4. I will simply try to deploy master on a test system and see where that leaves us.

**Comment by dennisnissle** [comment]
Hi there,

my bad, seems like the method is available starting with Bundles 6.11 (current ist 6.21.0).

> I tried replacing that file but other errors pop up where similar safe-guards are in place on master.

What does that mean exactly? Which errors?

**Comment by stylesuxx** [comment]
Sorry, I skipped a line in the stack trace. Replacing the file resolves our issues.
Will check why we are not seeing updates for bundles.

Thanks again for your quick help.
